contracts/StorageManager.sol

```diff
@@ -103,6 +100,7 @@ contract StorageManager {
     }
 
     /**
+    >> FOR PROVIDER
```

Comment:
Should we include a validation "require" after getting the Offer,  to ensure the Offer exists for the Sender? Otherwise ti will proceed and fail in the _setAvailableCapacity function if somebody that is not a Provider calls the function. Maybe its more elegant to have that validation. This impacts INCREASE, DECREASE and TERMINATE Offer.

Reply:
Sounds good, implemented this with modifier :-)